src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,62 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+
+    private K[] keyArray = (K[]) new Object[10];
+    private V[] valueArray = (V[]) new Object[10];
+    private int count = 0;
```

Comment:
```suggestion
    private K[] keys = (K[]) new Object[10];
    private V[] values = (V[]) new Object[10];
    private int size;
```
lets make number 10 as constant field https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names